plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/FeatureDevConstants.kt

```diff
@@ -3,7 +3,7 @@
 
 package software.aws.toolkits.jetbrains.services.amazonqFeatureDev
 
-const val FEATURE_NAME = "Amazon Q feature development"
+const val FEATURE_NAME = "Amazon Q Developer Agent for software development"
```

Comment:
why not just: `message("amazonqFeatureDev.code_generation.notification_title")`

Reply:
Notification title can change as per requirement, but `FEATURE_NAME` will always be constant for featureDev.